ocaml/xapi/message_forwarding.ml

```diff
@@ -2162,18 +2158,7 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
 
     let set_ssl_legacy ~__context ~self ~value =
       info "Host.set_ssl_legacy: host = '%s'; value = %b" (host_uuid ~__context self) value;
-      let success () =
-        if Db.Host.get_ssl_legacy ~__context ~self = value
-        then Some ()
-        else None
-      in
-      let local_fn = Local.Host.set_ssl_legacy ~self ~value in
-      let fn () =
-        do_op_on ~local_fn ~__context ~host:self
-          (fun session_id rpc ->
-             Client.Host.set_ssl_legacy rpc session_id self value)
-      in
-      tolerate_connection_loss fn success 30.
+      Local.Host.set_ssl_legacy ~__context ~self ~value
```

Comment:
Can't this just be a no-op?

Reply:
`Local.Host.set_ssl_legacy` is equivalent to `Xapi_host.set_ssl_legacy` which I have changed to be a no-op

Reply:
It can raise an exception, which is why I'm asking if it can be a no-op

Reply:
ah i see. well it could be, but a user would be confused if they see a `set_ssl_legacy ~value:true` command "succeed", but not actually do anything